test/app.js

```diff
@@ -42,7 +42,27 @@ const configService = {
   getValue: () => []
 };
 
-
-const metricTableContributor = new MetricsTableContributor('id', css, configService, 10, metricsTableConfig);
+// todo update test
+/**const metricTableContributor = new MetricsTableContributor('id', css, configService);
 metricTableContributor.configuration = metricsTableConfig;
-metricTableContributor.computeData(undefined);
+metricTableContributor.computeData(undefined);**/
+
+
+const testString1 = "new Date().getFullYear() === 2024;";
+const testString2 = "Math.abs(-5) + 10;";
+const testString3 = "\"\" + (new Date()).getTime() + ''";
+const testString4 = "'alphanumeric' + 123;";
+const testString5 = "test === \"example\";"; 
+const invalidTestString = "test === 'example';";  // This should fail because 'test' is not quoted
+const invalidTestString2 = "document.write('toto')";  // This should fail because 'test' is not quoted
+const invalidTestString3 = "window";  // This should fail because 'test' is not quoted
+
+
+console.log(processPassesAllowList(testString1, ''));
```

Comment:
It would be better if there is a real test applied or at least an additional comment like 'Should pass' or 'Should fail' and then the result

Reply:
i totally agree ! we will program that ;)